routers/repo/topic.go

```diff
@@ -27,6 +27,39 @@ func TopicPost(ctx *context.Context) {
 		topics = strings.Split(topicsStr, ",")
 	}
 
+	invalidTopics := make([]string, 0)
+	i := 0
+	for _, topic := range topics {
+		topic = strings.TrimSpace(strings.ToLower(topic))
+		// ignore empty string
+		if len(topic) > 0 {
+			topics[i] = topic
+			i++
+		}
+		if !models.TopicValidator(topic) {
+			invalidTopics = append(invalidTopics, topic)
+		}
+	}
+	topics = topics[:i]
+
+	if len(topics) > 25 {
+		log.Error(2, "Incorrect number of topics(max 25)")
```

Comment:
I don't think that this must be logged because the user receives the error message already. If you nevertheless want to log it, please use `log.Info` or `log.Warn` instead, since this is not a "real" error.

Reply:
That's reasonable, thank you